mapboxgl/viz.py

```diff
@@ -345,3 +360,88 @@ def add_unique_template_variables(self, options):
             lineWidth=self.line_width,
         ))
 
+
+class AltChoroplethViz(MapViz):
+    """Create a choropleth viz"""
+
+    def __init__(self,
+                 data,
+                 vector_polygon_source=False,
+                 vector_url=None,
+                 vector_layer_name=None,
+                 vector_join_color_property=None,
+                 join_data=[],
```

Comment:
Could this be merged with the class variable `data`?

Reply:
I did start this way initially but it added significant complexity/logic to the Jinja template. Seems like a good option if have two separate choropleth templates though!

Reply:
Yeah.  If it adds significant complexity to merge these into one, let's separate the `geojson choropleth` from the `vectortile choropleth` into two classes.  I'll leave this up to your judgment @akacarlyann

Reply:
With the separate html templates, this turned out to be simple so I've merged join_data with data in b11c006.